client/Scenarios/ShutdownVmsByLINQ.cs

```diff
@@ -15,9 +15,10 @@ public override void Execute()
             var client = new AzureResourceManagerClient();
             foreach (var sub in client.Subscriptions().List())
             {
```

Comment:
Why does the method take in a name filter but the call still has to use linq to filter the name?